cores/arduino/stm32/usb/cdc/usbd_cdc.h

```diff
@@ -41,9 +41,9 @@ extern "C" {
 /** @defgroup usbd_cdc_Exported_Defines
   * @{
   */
-#define CDC_IN_EP                                   0x81U  /* EP1 for data IN */
+#define CDC_IN_EP                                   0x82U  /* EP1 for data IN */
 #define CDC_OUT_EP                                  0x01U  /* EP1 for data OUT */
-#define CDC_CMD_EP                                  0x82U  /* EP2 for CDC commands */
+#define CDC_CMD_EP                                  0x83U  /* EP2 for CDC commands */

```

Comment:
ditto

Reply:
> Argh, just retry with BP the speed test... and got old results ⁉️
> 
> `Average bytes per second = 253013`

I test BluePill on the two platforms: OpenWRT MT7688 Linux host and on my MacBook Pro 2017... And i get about 760 000 bps on first and 720 000 on second.